proto/cosmos_proto/cosmos.proto

```diff
@@ -95,3 +101,11 @@ enum ScalarType {
     SCALAR_TYPE_STRING = 1;
     SCALAR_TYPE_BYTES = 2;
 }
+
+// SinceDescriptor describes the version at which a service, message or field was added.
+// Scalars extend simple protobuf built-in types with additional
+// syntax and semantics, for instance to represent versions.
+message SinceDescriptor {
+    // version is the version at which the field was added.
+    string version = 1;
+}
```

Comment:
What is this used for?

Reply:
This is the replacement of https://github.com/cosmos/cosmos-proto/pull/131. Currently, AutoCLI is supposed to filter out unsupported commands (because we release api from main), based on the since comment: 
https://github.com/cosmos/cosmos-sdk/blob/ed6b3ab/client/v2/internal/util/util.go#L39-L71
https://github.com/cosmos/cosmos-sdk/blob/ab919fed8db5272a7708e4caafc80de703d93bfc/client/v2/autocli/msg.go#L95-L97

However, that doesn't work because comments are always empty (https://github.com/cosmos/cosmos-proto/pull/133#issuecomment-1948397694). Once this is merged, I'll go through each since comment in the sdk and add the annotation, and then update autocli to use that annotation instead of checking the comments.